addons/mail_plugin/static/src/to_translate/translations_gmail.xml

```diff
@@ -43,6 +43,10 @@
     <string>Something bad happened. Please, try again later.</string>
     <string>Invalid URL</string>
     <string>No company attached to this contact.</string>
+    <string>Attachments could not be logged in Odoo because their total size exceeded the allowed maximum.</string>
```

Comment:
Why do you have 1 new entry in pot files and not any other, are they already existing ?